### ISSUE TITLE
Report -- separate the stats and plots into different sections for the test set and training set.

### ISSUE THREAD

**Issue by qchiujunhao** [goeckslab/Galaxy-Pycaret#48]
have separate tabs/sections for model performance on the train and test sets and plot confusion matrix + ROC AUC for both datasets.
